client/checkout/utils/upe.js

```diff
@@ -166,8 +166,8 @@ export const generateCheckoutEventNames = () => {
 		.join( ' ' );
 };
 
-export const appendPaymentMethodIdToForm = ( form, paymentMethodId ) => {
-	form.append(
+export const appendPaymentMethodIdToForm = ( $form, paymentMethodId ) => {
```

Comment:
Ensuring consistent naming convention - prepending the `$` in the name of the `form` argument, since it's a jQuery object